app/api/gov-holiday.js

```diff
@@ -1,42 +1,34 @@
-const Wreck = require('@hapi/wreck')
+const wreck = require('@hapi/wreck')
 
 async function getHolidayCalendarForEngland () {
-  const url = 'https://www.gov.uk/bank-holidays.json'
-  try {
-    const { payload } = await Wreck.get(url, { json: true })
-    if (!(payload?.['england-and-wales']?.events)) {
-      throw new Error('Invalid payload structure')
-    }
-    return payload['england-and-wales'].events // Returns only the events for England and Wales
-  } catch (err) {
-    console.error(`Getting holidays failed: ${err.message}`)
-    return [] // Return an empty array in case of error
+  const endpoint = 'https://www.gov.uk/bank-holidays.json'
+  const { payload } = await wreck.get(endpoint, { json: true })
+  if (!(payload?.['england-and-wales']?.events)) {
+    throw new Error('Invalid payload structure')
   }
+  return payload['england-and-wales'].events // Returns only the events for England and Wales
 }
 
 async function isTodayCustomHoliday () {
   const url = `${process.env.APPLICATION_API_URI}/holidays/isTodayHoliday`
   try {
-    await Wreck.get(url)
-    console.log('today is a custom holiday')
+    await wreck.get(url)
     return true
   } catch (err) {
-    console.error(`today is not a custom holiday : ${err.message}`)
-    return false
+    if (err.output.statusCode === 404) {
+      return false
+    } else {
+      throw err
+    }
   }
 }
 
 async function isTodayHoliday () {
-  try {
-    const holidays = await module.exports.getHolidayCalendarForEngland()
-    const today = new Date().toISOString().split('T')[0] // Format today's date as YYYY-MM-DD
-    let isHoliday = holidays?.some(holiday => holiday.date === today)
-    if (!isHoliday) isHoliday = await module.exports.isTodayCustomHoliday()
-    return isHoliday
-  } catch (err) {
-    console.error(`Checking holiday failed: ${err.message}`)
-    return false // Assume not a holiday in case of error
-  }
+  const holidays = await module.exports.getHolidayCalendarForEngland()
+  const today = new Date().toISOString().split('T')[0] // Format today's date as YYYY-MM-DD
+  let isHoliday = holidays?.some(holiday => holiday.date === today)
+  if (!isHoliday) isHoliday = await module.exports.isTodayCustomHoliday()
```

Comment:
I know you didn't write this line, was already here but I find this really confusing. Would prefer an if statement using curly braces as per others, but also just the repeated isHoliday makes it hard to parse `if (!isHoliday) isHoliday`. I had to triple take on that one

Reply:
I've been trying to avoid too much 'off topic' refactoring, which has been hard 😂, but as this line now has my name on it I'll make an exception 😉